docs/reference/migration/migrate_7_8.asciidoc

```diff
@@ -5,49 +5,65 @@
 ++++
 
 This section discusses the changes that you need to be aware of when migrating
-your application to Elasticsearch 7.8.
+your application to {es} 7.8.
 
 See also <<release-highlights>> and <<es-release-notes>>.
 
 coming[7.8.0]
 
+* <<breaking_78_mappings_changes>>
+* <<breaking_78_settings_changes>>
+
 //NOTE: The notable-breaking-changes tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
 
-//end::notable-breaking-changes[]
-
-[discrete]
-[[breaking_78_api_changes]]
-=== API changes
-
-[discrete]
-==== Deprecation of `local` parameter for get field mapping API
-
-The `local` parameter is a no-op and field mappings are always retrieved locally.
-As of 7.8, this parameter is deprecated and will be removed in next major version.
-
 [discrete]
 [[breaking_78_mappings_changes]]
 === Mappings changes
 
-[discrete]
 [[prevent-enabled-setting-change]]
-==== `enabled` setting cannot be changed on root mapper
+.The `enabled` mapping parameter cannot be changed for a root mapping.
+[%collapsible]
+====
+*Details* +
+Mapping requests that attempt to change the {ref}/enabled.html[`enabled`]
+mapping parameter for a root mapping will fail and return an error.
+
+Previously, {es} accepted mapping requests that attempted to change the
+`enabled` parameter of the root mapping. Theses changes were not applied, but
+such requests didn't return an error.
+
+*How to migrate* +
+To avoid errors, do not submit mapping requests that change the
+{ref}/enabled.html[`enabled`] mapping parameter.
+====
 
-Previously, Elasticsearch accepted mapping updates that tried to change the
-`enabled` setting on the root mapping. The update was not applied, but the
-request didn't throw an error. As of 7.8, requests that attempt to change
-`enabled` on the root mapping will fail.
+.The get field mapping API's `local` query parameter is deprecated.
+[%collapsible]
+====
+*Details* +
+The {ref}/indices-get-field-mapping.html[get field mapping API]'s `local` query
+parameter is deprecated and will be removed in {es} 8.0.0.
+
+The `local` parameter is a no-op. The API always retrieves field mappings
+locally.
+
+*How to migrate* +
+To avoid deprecation warnings, discontinue use of the `local` parameter.
+====
 
 [discrete]
 [[breaking_78_settings_changes]]
 === Settings changes
 
-[discrete]
 [[deprecate-node-local-storage]]
-==== `node.local_storage` is deprecated
+.The `node.local_storage` setting is deprecated
+[%collapsible]
+====
+The `node.local_storage` setting is deprecated. In {es} 8.0.0, all nodes require
```

Comment:
IMO it looks odd that this change doesn't have the same sections as the others.  I don't know this setting, so I'm left wondering what I need to do about this change. Do I have to take action or is this just informational?

Reply:
I added a **Details** and **Impact** section with 20db1c358f3.

I was originally considering leaving off **Impact** as this setting was previously undocumented and unlikely to be used. However, adding a discontinue instruction doesn't seem harmful.

I guess in situations where we don't have an actionable advice, we could say something along the lines of "No action needed." under **Impact**.